gpustack/envs/__init__.py

```diff
@@ -41,3 +41,8 @@
     "true",
     "1",
 ]
+
+# External authentication defaults
+EXTERNAL_AUTH_DEFAULT_ACTIVE = os.getenv(
+    "GPUSTACK_EXTERNAL_AUTH_DEFAULT_ACTIVE", "false"
+).lower() in ["true", "1"]
```

Comment:
We can remove this part. These envs are arguments not exposed in GPUStack config.